modular_skyrat/modules/supersoups/code/soup_mixtures.dm

```diff
@@ -0,0 +1,12 @@
+/datum/chemical_reaction/food/soup/clean_up(datum/reagents/holder, datum/equilibrium/reaction, react_vol)
+    . = ..()
+    if(!length(results))
```

Comment:
Yeah, alt click let's you set volume transfer.

Reply:
> Is alt already being used by something? There's also the less-used `alt_click_secondary()` (alt-right-click). You could make it an action button, or make it toggled by attacking it with an item...lot of possibilities.

I'm currently working on that fix as I couldn't sleep because it's irking me. How does one apply something like this. If it's easier for you, my discord is the same as my username here